frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/storage-tab-validation.ts

```diff
@@ -28,6 +28,8 @@ export const validateStorages = (options: UpdateOptions) => {
   }
 
   const storages = getStoragesWithWrappers(state, id);
+  const templateValidation = getTemplateValidation(state, id);
+  const allowedBusses = templateValidation ? templateValidation.getAllowedBusses() : null;
```

Comment:
the default is all right? Let's rather do this

```suggestion
  const allowedBusses = (templateValidation || new TemplateValidations()).getAllowedBusses();
```